scripts/tests/api_compare/filter-list

```diff
@@ -7,3 +7,4 @@
 # https://github.com/ChainSafe/forest/issues/4661
 !Filecoin.EthGetBlockByNumber
 !eth_getBlockByNumber
+!Filecoin.EthGetTransactionReceipt
```

Comment:
Is there a tracking issue for re-enabling tests for this method?

Reply:
I've added this subtask to this tracking issue: https://github.com/ChainSafe/forest/issues/4759